packages/status-react/src/routes/index.tsx

```diff
@@ -57,20 +56,22 @@ export const Community = (props: Props) => {
       <ProtocolProvider options={{ publicKey, environment }}>
         <AppProvider options={options}>
           <DialogProvider>
-            <GlobalStyle />
-            <Wrapper>
-              <MainSidebar />
-              <Routes>
-                <Route
-                  path="/:id"
-                  element={
-                    <Gate>
-                      <Chat />
-                    </Gate>
-                  }
-                />
-              </Routes>
-            </Wrapper>
+            {/* todo?: wrap inside another element to simulate `html` > `body` structure */}
+            <div className={base()}>
```

Comment:
This could also be a styled component directly.

Reply:
Applied to the wrapper.

Also, you were right, the base styles worked in the styled component. Let me know if I should move it to the `Wrapper` after all.